### PR TITLE
CF-562 Add debug logs to generate_load.

### PR DESCRIPTION
**Pull request by plomakin**
Added generate_load.log file and log handler.
Moved couple of constants to config.py
Extended logging in generate_load and base.py

### CODE REVIEW

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

File: cloudferry_devlab/cloudferry_devlab/generate_load.py [hunk 1]
```diff
@@ -927,6 +974,7 @@ def break_vm(self, vm_id):
         """ Method delete vm via virsh to emulate situation, when vm is valid
         and active in nova db, but in fact does not exist
         """
+        self.log.debug("Breaking VM with id %s.", vm_id)
```
Comment (roman-verchikov) [inline cloudferry_devlab/cloudferry_devlab/generate_load.py]: "Breaking VM..." is not clear, I'd suggest adding debug log of `virsh destroy && virsh undefine` command in addition to this message

**Comment by roman-verchikov** [comment]
-1, please resolve minor comment

File: cloudferry_devlab/cloudferry_devlab/generate_load.py (hunk 1 above)
Comment (plomakin) [inline cloudferry_devlab/cloudferry_devlab/generate_load.py]: will do

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success
http://jenkins-cf.vm.mirantis.net:8080/job/cloudferry-unittests-and-style-checks/3838/

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

**Comment by roman-verchikov** [comment]
+1

File: cloudferry_devlab/cloudferry_devlab/generate_load.py [hunk 2]
```diff
@@ -95,11 +91,15 @@ def get_params_for_user_creating(_user):
...
             users = self.config.users
         for user in users:
+            self.log.debug("Creating user with name %s.", user['name'])
             self.keystoneclient.users.create(
                 **get_params_for_user_creating(user))
             if not user.get('additional_tenants'):
                 continue
             for tenant in user['additional_tenants']:
+                msg = "Adding user {} with role {} to tenant {}.".format(
+                    user['name'], tenant['role'], tenant['name'])
```
Comment (antonf) [inline cloudferry_devlab/cloudferry_devlab/generate_load.py]: Please don't use "...".format(x, y, z) for logging. Use `self.log.debug("Adding user %s with role %s to tenant %s.", user['name'], tenant['role'], tenant['name'])` for formatting.

**Comment by antonf** [comment]
👎

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

File: cloudferry_devlab/cloudferry_devlab/tests/utils.py [hunk 3]
```diff
@@ -156,9 +160,13 @@ def execute_command_on_vm(self, ip_addr, cmd, username=None,
                                  warn_only=warn_only):
             try:
                 if use_sudo:
-                    return fabric_api.sudo(cmd, shell=False)
+                    output = fabric_api.sudo(cmd, shell=False)
+                    self.log.debug(output)
+                    return output
                 else:
-                    return fabric_api.run(cmd, shell=False)
+                    output = fabric_api.sudo(cmd, shell=False)
```
Comment (roman-verchikov) [inline cloudferry_devlab/cloudferry_devlab/tests/utils.py]: Should be `fabric_api.run` here. Also make sure the logs clearly show whether the call was done with `sudo` or with `run`

**Comment by roman-verchikov** [comment]
-1

File: cloudferry_devlab/cloudferry_devlab/tests/utils.py (hunk 3 above)
Comment (plomakin) [inline cloudferry_devlab/cloudferry_devlab/tests/utils.py]: wow, that's a big mistake. sorry will fix it.

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

**Comment by MirantisWorkloadMobility** [comment]
There were no python files updated. Admins, please, check pull request.

**Comment by plomakin** [comment]
retest

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

**Pull request merged**